test/WebSites/FormatFilterWebSite/project.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+    "commands": {
+        "web": "Microsoft.AspNet.Hosting server=Microsoft.AspNet.Server.WebListener server.urls=http://localhost:5001",
+        "kestrel": "Microsoft.AspNet.Hosting --server Kestrel --server.urls http://localhost:5000"
+    },
+    "dependencies": {
+        "Kestrel": "1.0.0-*",
+        "Microsoft.AspNet.Mvc": "6.0.0-*",
+        "Microsoft.AspNet.Mvc.TestConfiguration": "1.0.0",
+        "Microsoft.AspNet.Server.IIS": "1.0.0-*",
+        "Microsoft.AspNet.Server.WebListener": "1.0.0-*",
+        "Microsoft.AspNet.StaticFiles": "1.0.0-*"
+    },
+    "frameworks": {
+        "aspnet50": { },
+        "aspnetcore50": { }
+    },
+    "webroot": "wwwroot"
```

Comment:
The final decision on this is we are going to have a HelloWorld.html with content HelloWorld in wwwroot and a readme.md directly in the project. The reason for that is that otherwise readme.md does not show up in the website. 
I have a workitem to move all the existing readme.md one folder up and add HelloWorld.html in place of that.